style/tailwind.css

```diff
@@ -1,3 +1,141 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.markdown-content {
+  @apply bg-base-200 mt-4 p-5 rounded space-y-4;
+}
+
+.markdown-content h1 {
+  @apply text-2xl font-black;
```

Comment:
`font-black` should probably be removed, as it might not work well with a dark theme of daisy-ui.

Reply:
Despite the name, `font-black` is setting the font weight, not the text color. https://tailwindcss.com/docs/font-weight

Reply:
Ah cool.